packages/web/src/components/link/UserLink.tsx

```diff
@@ -49,15 +49,33 @@ export const UserLink = (props: UserLinkProps) => {
     return null
   }
 
-  // Prepare the user badges using the new TieredUserBadge component
+  // Prepare the user badges
   const badges = noBadges ? null : (
-    <TieredUserBadge
+    <UserBadges
       userId={userId}
       size={badgeSize}
       css={{ marginTop: spacing['2xs'] }}
     />
   )
 
+  // Create a text element that may or may not be wrapped in a popover
```

Comment:
just checking this is a pure refactor? it is cleaner, just wanted to check though since this component is used everywhere